File: java/graphql-client/src/main/java/com/sdl/web/pca/client/GraphQLError.java
package com.sdl.web.pca.client;

import java.util.List;

public class GraphQLError {
    public String Message;
    public List<Object> Path;
    public Object locations;
    public Object errorType;
    public Object extensions;

    public void setMessage(String message) {
        Message = message;
    }

    public String getMessage() {
        return Message;
    }

    public void setPath(List<Object> path) {
        Path = path;
    }

    public List<Object> getPath() {
        return Path;
    }
}

File: net/src/Sdl.Tridion.Api.Client/GraphQLClient/Response/GraphQLError.cs
using System.Collections.Generic;

namespace Sdl.Tridion.Api.GraphQL.Client.Response
{
    /// <summary>
    /// GraphQL Error holds error information regarding the failed request.
    /// </summary>
    public class GraphQLError
    {
        /// <summary>
        /// Error message.
        /// </summary>
        public string Message { get; set; }

        /// <summary>
        /// Path to the error within the GraphQL query.
        /// </summary>
        public List<object> Path { get; set; }

        /// <summary>
        /// Locations of all errors within the GraphQL query.
        /// </summary>
        public List<GraphQLErrorLocation> Locations { get; set; }

        /// <summary>
        /// GraphQL extensions.
        /// </summary>
        public GraphQLExtensions Extensions { get; set; }
    }
}
